backend/helpers/helpers.go

```diff
@@ -140,10 +50,7 @@ func ParseInt(w http.ResponseWriter, r *http.Request, key string) (error, uint)
 	return nil, uint(intKey)
 }
 
-func GetAllIdsFromList[T models.AllTables](structList []T) []uint {
-	var list []uint
-	for _, element := range structList {
-		list = append(list, element.GetId())
-	}
-	return list
+func FormatEmail(email string) string {
+	x := strings.Split(email, "@")
+	return strings.ToLower(strings.Replace(x[0], ".", "", -1) + "@" + x[1])
```

Comment:
im so confused so what is this going to do with my email a.barroso2001@gmail.com?

Reply:
https://support.google.com/mail/answer/7436150?hl=en#:~:text=Adding%20dots%20doesn't%20change,own%20johnsmith%40gmail.com.

Reply:
From my understand, dots aren't a real thing in email addresses but rather a convenience. Same with caps

Reply:
I just don't see why we should change what the user entered as their email, like caps is nice sure, but otherwise what benefit does removing the dot give us?

Reply:
> https://support.google.com/mail/answer/7436150?hl=en#:~:text=Adding%20dots%20doesn't%20change,own%20johnsmith%40gmail.com.

The dot rule only applies to gmail and a few other providers. This is not a universal rule, so we should not be applying it here. We should use whatever the user inputted to not accidentally break something that uses custom routing rules.

Case-insensitivity is universal to all email servers though, so I think it's fine to keep that.

Reply:
ok, ill fix that when I have time